_posts/2019-03-04-jumping-on-the-dash-less-bandwagon.markdown

```diff
@@ -0,0 +1,64 @@
+
+**Link of Interest:** [Contemporary Views on Serverless and Implications](https://m.subbu.org/contemporary-views-on-serverless-and-implications-1c5907c611d8)
+
+## Business Impacts
+
+If you’re non-technical then you are probably wondering why you should care about Serverless. After all, isn't it just another buzzword? Actually, Serverless is going to have more of an impact on how you work and operate your business than it will for the techies implementing it. Speed, visibility and agility is what it’s all about.
+
+Serverless is about speed. In my time at Mechanical Rock, I have personally received an end-to-end feature request (web, backing code and accompanying infrastructure) in the morning and delivered it to production fully tested by lunch time. A by-product of speed is the ability to experiment. Why try out just one idea, when it's possible to try out multiple ideas, gather the results and then implement the best bits?
+
+Serverless grants unrivalled visibility into the cost and profitability of your business at an incredibly fine-grained level. You can literally see how much it costs to provide the ‘Contact us’ form on your website. Don’t believe me? Read the thought-provoking article at the end of this section.
+
+Serverless agility is about the highlighting (and hopefully the removal) of waste. Anything that gets in the way of delivering value is immediately highlighted, so if it is a technical hurdle then go around it; if its operational then work out how to kill it or change it. End customers don’t care about what bureaucracy you think you need, they’ll vote with eyeballs and wallets and just go somewhere else if you can’t deliver the goods.
+
+**Link of Interest:** [Micro-Optimization: Activity-Based Costing for Digital Services?](https://aws.amazon.com/blogs/enterprise-strategy/micro-optimization-activity-based-costing-for-digital-services/)
+
+## Technical Impacts
+
+For all you technical readers out there, be afraid of serverless. It isn’t a flash in the pan buzzword that’s going away. It is coming for you and will make you obsolete if you don’t adapt. The full power of serverless requires a different way of working and thinking.
+
+Firstly, it removes the infrastructure downtime between tasks. In the past you’d have to wait for servers to become available. Then you need to wait for software (like application servers) to be installed and configured. Then it was gaining database access for your code. Etcetera etcetera etcetera. With serverless, once you have your access permissions, you can have a brand-new application, in production, within hours. Everything needed to host and operate your application is provided to you by the cloud provider.
```

Comment:
>Etcetera etcetera etcetera.

No need for this.

Reply:
>you can have a brand-new application, in production, within hours

Suggest:
>you can have a brand-new application, in production, within minutes or hours depending on architectural complexity.

Reply:
Replaced:
Etcetera etcetera etceterwith